packages/components/src/Tabs/Tabs.tsx

```diff
@@ -38,21 +33,38 @@ export const Tabs = ({
     size,
     variant,
   });
-
   return (
     <TabContext.Provider value={{ classNames }}>
-      <div
-        className={cn('flex', gapSpace[space], classNames.tabs)}
-        {...tabListProps}
-        ref={ref}
-      >
-        {[...state.collection].map(item => {
-          return <Tab key={item.key} item={item} state={state} />;
-        })}
+      {/* tabs container */}
+      <div className={className}>
```

Comment:
this should be possible to be styled via theme? :) Can you extend the component style thing?